Source/ChocolateyGui/Views/RemoteSourceView.xaml

```diff
@@ -20,31 +19,43 @@
+                     VerticalContentAlignment="Center"
+                     controls:TextBoxHelper.Watermark="{x:Static lang:Resources.RemoteSourceView_SearchBoxText}"
                      Text="{Binding SearchQuery, UpdateSourceTrigger=PropertyChanged}" />
-            <CheckBox Name="AllVersionsCheckBox" Margin="10,5,5,5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxAllVersions}"
+                </StackPanel>
+                <CheckBox Name="AllVersionsCheckBox" Margin="5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxAllVersions}"
                       IsChecked="{Binding IncludeAllVersions}" />
-            <CheckBox Name="PrereleaseCheckBox" Margin="5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxIncludePrerelease}"
+                <CheckBox Name="PrereleaseCheckBox" Margin="5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxIncludePrerelease}"
                       IsChecked="{Binding IncludePrerelease}" />
-            <CheckBox Name="MatchCheckBox" Margin="5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxMatchExactly}" IsChecked="{Binding MatchWord}" />
-            <ComboBox Name="SortBox" Margin="5" ItemsSource="{Binding SortOptions}" SelectedValue="{Binding SortSelection}"/>
-        </StackPanel>
-        <StackPanel Grid.Row="0" Orientation="Horizontal" HorizontalAlignment="Right" VerticalAlignment="Center"
+                <CheckBox Name="MatchCheckBox" Margin="5" Content="{x:Static lang:Resources.RemoteSourceView_CheckboxMatchExactly}" IsChecked="{Binding MatchWord}" />
+                <ComboBox Name="SortBox" Margin="5" ItemsSource="{Binding SortOptions}" SelectedValue="{Binding SortSelection}"/>
+            </WrapPanel>
+            <StackPanel Grid.Column="1" Orientation="Horizontal" HorizontalAlignment="Right" VerticalAlignment="Top"
                     Margin="10,5">
-            <Button Command="{commands:DataContextCommandAdapter RefreshRemotePackages, CanRefreshRemotePackages}"
-                    Style="{StaticResource SquareButtonStyle}" Content="{x:Static lang:Resources.RemoteSourceView_ButtonRefreshPkgs}" Margin="0,4,0,4" />
-        </StackPanel>
+                <Button Command="{commands:DataContextCommandAdapter RefreshRemotePackages, CanRefreshRemotePackages}"
+                    Style="{StaticResource SquareButtonStyle}" ToolTip="{x:Static lang:Resources.RemoteSourceView_ButtonRefreshPkgs}" Margin="0,4,0,4">
+                    <iconPacks:PackIconFontAwesome Kind="Refresh" />
+                </Button>
+            </StackPanel>
+        </Grid>
 
-        <Grid Grid.Row="2" DockPanel.Dock="Bottom" Margin="4">
+        <Grid Grid.Row="2" Margin="4" Grid.IsSharedSizeScope="True">
             <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="*" />
-                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="Auto" SharedSizeGroup="ButtonCol" />
```

Comment:
Seriously though, where has this `SharedSizeGroup` been all my life?

Reply:
In this case it's a cool feature. But sometimes it can be in some other cases a performance impact for the UI, so it should be used carefully.